mongo/crypt_retrievers.go

```diff
@@ -14,10 +14,14 @@ import (
 
 // keyRetriever gets keys from the key vault collection.
 type keyRetriever struct {
-	coll *Collection
+	coll                    *Collection
+	supportExplicitSessions bool
```

Comment:
Since we're opting _not_ to support explicit sessions for these operations right now, what is the point of `supportExplicitSessions` here?

Reply:
Oops, removed. That was a relic from an earlier experiment.